web/client/themes/default/less/widget.less

```diff
@@ -9,6 +9,12 @@
 // **************
 // Theme
 // **************
+.setFontSize (@val){
+    @fSize: @val
+};
+.em (@prop, @val) {
```

Comment:
what if we want to use this function somewhere else?
is there a way to include it in the mapstore theme?
like this color-var in css-properties.less?

https://github.com/geosolutions-it/MapStore2/blob/7567d12b49fea8a00ac827b77708856e9927b125/web/client/themes/default/less/mixins/css-properties.less#L15-L21